### DISCUSSION TITLE
Outdated SSL certificate in a new project

### DISCUSSION THREAD

**Discussion by pasierb** [supabase/supabase#16886] [Questions]
Hey Supabase fam!

I'm having an issue with outdated SSL certificate in a project I set up couple day ago on a free plan. Any known way to regenerate SSL cert?

<img width="906" alt="Screenshot 2023-08-28 at 00 18 05" src="https://github.com/supabase/supabase/assets/357043/61cc04d7-73a3-42e3-a99c-d064106e28b9">

**Comment by GaryAustin1** [comment]
You should contact support.
